pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+###############################################################################
+[tool.poetry]
+name = "smqtk_relevancy"
+version = "0.15.0"
+description = "SMQTK Relevancy"
+authors = ["Kitware, Inc. <smqtk-developers@kitware.com>"]
+license = "BSD-3-Clause"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+setuptools = ">=53"
+numpy = "^1.19.5"
+pillow = "^8.2.0"
+smqtk-core = "^0.17.0"
+smqtk-dataprovider = "^0.15.0"
+smqtk-classifier = "^0.15.0"
```

Comment:
Lets use to `>=` qualifier for our `0.*` minor versions (`^0.17.0` specifically bounds within the `0.17.*` version range), and update to versions 0.18.0, 0.16.0 and 0.16.0 respectively. Indexing package version update pending my finishing of Kitware/SMQTK-Indexing#8